net/pfSense-pkg-freeradius2/Makefile

```diff
@@ -56,6 +59,10 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}${DATADIR}/info.xml \
 		${STAGEDIR}${DATADIR}
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/pkg/googleauth \
+		${STAGEDIR}/usr/local/etc/raddb/modules
+	${INSTALL_SCRIPT} ${FILESDIR}${PREFIX}/pkg/googleauth.py \
+		${STAGEDIR}/usr/local/etc/raddb/scripts
```

Comment:
Replace `/usr/local` by `${PREFIX}`

Also please add the new files on correspondent path inside files dir. Instead of add new file to `files/usr/local/pkg`, add them to `files/usr/local/etc/raddb`